packages/frontend/src/main/components/viewer/FilterNumericActive.vue

```diff
@@ -104,59 +115,66 @@ export default {
     active: { type: Boolean, default: false },
     preventFirstSet: { type: Boolean, default: false }
   },
+  setup() {
+    const { result: viewerStateResult } = useQuery(gql`
+      query {
+        commitObjectViewerState @client {
+          currentFilterState
+          objectProperties
+          localFilterPropKey
+        }
+      }
+    `)
+    const viewerState = computed(
+      () => viewerStateResult.value?.commitObjectViewerState || {}
+    )
+
+    return { viewerState }
+  },
   data() {
     return {
       range: [0, 1],
       colorBy: true,
       width: 300,
-      preventFirstSetInternal: this.preventFirstSet
+      preventFirstSetInternal: this.preventFirstSet,
+      userMin: 0,
+      userMax: 1,
+      error: false,
+      errorMessages: []
     }
   },
   watch: {
     filter(newVal) {
-      this.$set(this.range, 0, newVal.data.minValue)
-      this.$set(this.range, 1, newVal.data.maxValue)
-    },
-    colorBy() {
-      this.setFilter()
+      this.$set(this.range, 0, newVal.min)
+      this.$set(this.range, 1, newVal.max)
     }
   },
   mounted() {
-    this.$set(this.range, 0, this.filter.data.minValue)
-    this.$set(this.range, 1, this.filter.data.maxValue)
-    this.setFilter()
+    this.$set(this.range, 0, this.filter.min)
+    this.$set(this.range, 1, this.filter.max)
+
+    setTimeout(() => {
```

Comment:
ditto re: timeouts. is there some sort of event we could use here instead?

Reply:
The case where this happens: we're setting a filter directly from a URL/comment with separate passMin and passMax values (e.g., `area > 4 && area < 10`). 

There's no way to update that component besides via $refs (that I found); it looks like because of the dynamic imports the `this.$refs.histogramSlider` is `undefined` in mounted. Keeping those webpack chunks works with the timeout. 

Removing the dynamic imports works with a slightly less hacky `$nextTick` call in mounted. I'll push these changes now.